Formula/x/xrick.rb

```diff
@@ -5,6 +5,9 @@ class Xrick < Formula
   # There is a repo at https://github.com/zpqrtbnk/xrick but it is organized
   # differently than the tarball
   sha256 "aa8542120bec97a730258027a294bd16196eb8b3d66134483d085f698588fc2b"
+  # From README:
+  # 'So the code is released "in the spirit" of the GNU GPL. Whatever that means.'
+  license :cannot_represent
```

Comment:
No idea what to do here. Full section in README says:
```
I have written the xrick code. However, graphics and maps and sounds
are by the authors of the original Rick Dangerous game, and "Rick
Dangerous" remains a trademark of its owner(s) -- maybe Core Design
(who wrote the game) or FireBird (who published it). As of today, I
have not been successful at contacting Core Design.

This makes it a bit difficult to formally release the whole code,
including data for graphics and maps and sounds, under the terms of
licences such as the GNU General Public Licence. So the code is
released "in the spirit" of the GNU GPL. Whatever that means.
```

There are various repos that do have `xrick` - https://repology.org/project/xrick/versions

Reply:
I believe that means this formula contains copyrighted contents that we can't distribute freely (under a open-source license), so I'm leaning towards removing it instead.

Reply:
Yeh, sadly should probably remove, agreed.

Reply:
Okay. Luckily the install count is low:
```
==> Analytics
install: 1 (30 days), 2 (90 days), 97 (365 days)
install-on-request: 1 (30 days), 2 (90 days), 97 (365 days)
build-error: 0 (30 days)
```